SwiftCompilerSources/Sources/Optimizer/ModulePasses/MandatoryPerformanceOptimizations.swift

```diff
@@ -47,6 +47,15 @@ let mandatoryPerformanceOptimizations = ModulePass(name: "mandatory-performance-
   }
 }
 
+/// Performance the same set of top-down optimization as the above "mandatory-performance-optimizations"
+/// pass, but only on the initializers of '@const' globals.
+let constGlobalVariableFoldingPass = ModulePass(name: "const-global-variable-folding") {
```

Comment:
Do you really need a new pass for this? Why not adding those init functions in the MandatoryPerformanceOptimization pass?

Reply:
Moved collection to `MandatoryPerformanceOptimization`.